justice/align_model/synthetic_positives.py

```diff
@@ -64,13 +65,19 @@ def __init__(
     def make_xform(self):
         translate_time = self.rng.normalvariate(0, self.translate_time_stdev)
         translate_flux = self.rng.normalvariate(0, self.translate_flux_stdev)
-        log_dilate_time = self.rng.normalvariate(0, self.log_dilate_time_stdev)
-        log_dilate_flux = self.rng.normalvariate(0, self.log_dilate_flux_stdev)
+
+        dilations = truncnorm.rvs(
+            math.log(0.04),
+            math.log(25),
```

Comment:
looks good, but i think the point is to decrease this ... so maybe either add a parameter like `dilate_time_max_factor` / `dilate_flux_max_factor`, or specify standard deviations. min should always be negative of max.